tests/tests.py

```diff
@@ -0,0 +1,97 @@
+import json
+from unittest import expectedFailure
+
+from django.core.urlresolvers import reverse, resolve
+from django.test import SimpleTestCase, RequestFactory
+
+from edtf.views import edtf_form, boolean_result, result_json
+
+
+class TestURLs(SimpleTestCase):
+
+    def test_edtf_form_url(self):
+        view = resolve(reverse('edtf:edtf_form')).func
+        self.assertEqual(view, edtf_form)
+
+    def test_results_json_url(self):
+        view = resolve(reverse('edtf:results_json')).func
+        self.assertEqual(view, result_json)
+
+
+class TestEdtfFormView(SimpleTestCase):
+
+    def setUp(self):
+        self.factory = RequestFactory()
+
+    def test_returns_200(self):
+        request = self.factory.get('/')
+        response = edtf_form(request)
+        self.assertEqual(response.status_code, 200)
+
+    def test_correct_template_used(self):
+        url = reverse('edtf:edtf_form')
+        response = self.client.get(url)
+        self.assertTemplateUsed(response, 'edtf/edtf_form.html')
+
+
+class TestBooleanResultView(SimpleTestCase):
+
+    def setUp(self):
+        self.factory = RequestFactory()
+
+    def test_returns_200(self):
```

Comment:
I would *love* to get rid of those views. And yeah, I didn't put too much work into the tests for the views since I thought it was very likely they might go away. They were pretty much copy-pasted from the result_json tests. I'll delete the mentioned views and url stuff, as well as these tests.